test/benchmark_http_client_test.cc

```diff
@@ -276,8 +297,38 @@ TEST_F(BenchmarkClientHttpTest, BadContentLength) {
   };
 
   EXPECT_CALL(stream_encoder_, encodeData(_, _)).Times(0);
-  testBasicFunctionality(1, 1, 1);
+  testBasicFunctionality(1, 1, 1, request_generator);
   EXPECT_EQ(1, getCounter("http_2xx"));
 }
-
+TEST_F(BenchmarkClientHttpTest, MultipleRequestsDifferentPath) {
```

Comment:
preferably start method names with a verb

Reply:
I'm not sure what this comment is applying to. Is this the MultipleRequestsDifferentPath name?

Reply:
test names names should be action-condition-result.

Reply:
done.